aliBuild

```diff
@@ -237,10 +247,10 @@ class HttpRemoteSync:
       else:
         hashList = json.loads(urlopen(hashListUrl).read())
         pkgList = json.loads(urlopen(pkgListUrl).read())
-    except URLError, e:
+    except URLError as e:
```

Comment:
Ah, I see; just to make merging easier down the line, not for simultaneous py3/py2.4 compatibility, which I don't think can happen easily.
Yeah I can create a new branch/pull request & cherry pick compatible changes. Should I also rebase this branch? I don't know if rebasing will break the pull request.